backend/match4everyone/configuration/B.py

```diff
@@ -13,8 +13,8 @@ class BConfig(ParticipantConfig):
     #    NewPermissions.can_contact_type_a_if_approved,
     # ]
 
-    profile_visible_for_A = True
-    profile_visible_for_other_B = False
+    profile_visible_for_participants_of_different_type = True
+    profile_visible_for_participants_of_same_type = False
 
```

Comment:
Is this missing a
```suggestion
    needs_manual_approval_from_staff = True
```
line? :)

Reply:
I set the default value to `True` in the superclass to make approvals the default behaviour, so it is not necessary.

Reply:
I think I would still put it here for a better understanding where to change it, don't you think?